face/UnifiedFaceCollection/python/unified_face_collection.py

```diff
@@ -1,117 +1,131 @@
+            print(f"Large Face List: {self.large_face_list_id} already exists.")
+        except Exception:
+            print(f"Creating Large Face List: {self.large_face_list_id}")
+            self.face_admin_client.large_face_list.create(
+                self.large_face_list_id,
+                name=self.face_collection_id + " Face List",
+                recognition_model=FaceRecognitionModel.RECOGNITION04
+            )
         
         # Create Large Person Group
-        person_group_url = f"{self.face_api_url}/largepersongroups/{self.large_person_group_id}"
-        person_group_response = requests.put(person_group_url, headers=self.headers, json={"name": self.face_collection_id + " Person Group", "recognitionModel": "recognition_04"})
-        if person_group_response.status_code == 200:
-            print(f"Person group {self.large_person_group_id} created successfully.")
-        else:
-            print(person_group_response.json())
+        try :
+            self.face_admin_client.large_person_group.get(self.large_person_group_id)
+            print(f"Large Person Group: {self.large_person_group_id} already exists.")
+        except Exception:
+            print(f"Creating Large Person Group: {self.large_person_group_id}")
+            self.face_admin_client.large_person_group.create(
+                self.large_person_group_id,
+                name=self.face_collection_id + " Person Group",
+                recognition_model=FaceRecognitionModel.RECOGNITION04
+            )
 
     def add_face(self, image_path, person_name=None):
-        add_face_headers = {
-            'Ocp-Apim-Subscription-Key': self.subscription_key,
-            'Content-Type': 'application/octet-stream',
-            'X-MS-AZSDK-Telemetry': self.injection_header
-        }
-        params = {
-            'detectionModel': 'detection_03'
-        }
-        
-        faces = detect_faces(self.subscription_key, self.endpoint, image_path, self.injection_header)
+        faces = self.detect_faces(image_path)
+        target_face = None
         if len(faces) == 0:
             return "No faces detected in the image."
         elif len(faces) > 1:
-            image_width, image_height = get_image_dimensions(image_path)
+            image_width, image_height = self.get_image_dimensions(image_path)
             print(f"Multiple faces detected. Using the first face (largest face) for adding to the collection.")
-            face_rectangle = enlarge_bounding_box(faces[0]['faceRectangle'], image_width, image_height)
-            params['targetFace'] = f"{face_rectangle['left']},{face_rectangle['top']},{face_rectangle['width']},{face_rectangle['height']}"
+            face_rectangle = self.enlarge_bounding_box(faces[0]['faceRectangle'], image_width, image_height)
+            target_face = [face_rectangle['left'],face_rectangle['top'],face_rectangle['width'], face_rectangle['height']]
```

Comment:
[nit] add extra space

```suggestion
            target_face = [face_rectangle['left'], face_rectangle['top'], face_rectangle['width'], face_rectangle['height']]
```